src/components/Sidebar.tsx

```diff
@@ -38,10 +39,10 @@ const Sidebar: React.FC<{ steps: { title: string; link: string }[] }> = ({
         </HelperIcon>
         <HelperText>
           Welcome to the Learning Pathway! Use the sidebar to follow the guide.
-          Open the{" "}
+          Open the
           <Link to="/" target="_blank" rel="noopener noreferrer">
             Template Playground
-          </Link>{" "}
```

Comment:
This change affected the white space in the text here. Please fix.
<img width="247" alt="Screenshot 2025-03-13 at 5 18 53 PM" src="https://github.com/user-attachments/assets/9036e0ac-04ef-4c7c-b608-e99c39d99e2f" />